lib/virtio/virtqueue.c

```diff
@@ -231,6 +231,63 @@ void *virtqueue_get_available_buffer(struct virtqueue *vq, uint16_t *avail_idx,
 	return buffer;
 }
 
+void *virtqueue_get_next_avail_buffer(struct virtqueue *vq, uint16_t idx,
```

Comment:
do you still need this one? virtqueue_get_available_buffers should replace this function